align/compiler/match_graph.py

```diff
@@ -120,8 +121,8 @@ def _remove_group_const(self, subckt, rm_const):
                 subckt.constraints.remove(const)
             assert len(subckt.constraints) == start_count - len(rm_const)
 
-    def _group_block_const(self, name):
-        subckt = self.ckt_data.find(name)
+    def _group_block_const(self, sname):
```

Comment:
Variable names could be refactored to improve readability. For example `subcircuit_name` instead of `sname`. `instance_name` instead of `gb_name` or else.

Reply:
done